static/components/List.js

```diff
@@ -1,78 +1,85 @@
 import React, { Component } from 'react';
-import ListItem from './listitem';
-import uuidv4 from 'uuid/v4';
+import ListItem from './ListItem';
+import uuidv4 from 'uuid/v4'; // generate unique id
 
 class List extends Component {
     constructor(props) {
         super(props);
 
         this.state = {
-            items: []
+            items: [],
         };
-        this.removeItem = this.removeItem.bind(this)
+        this.removeItem = this.removeItem.bind(this);
```

Comment:
Are you sure you need context binding here? The purpose of using arrow functions instead of class methods syntax was to avoid  context binding as far as I remember